airflow/providers/databricks/hooks/databricks.py

```diff
@@ -379,20 +379,27 @@ def import_notebook(self, notebook_name: str, raw_code: str, language: str, over
             raise ValueError(f"results: format must be one of the following: {str(format_options)}")
 
         # encode notebook
-        encodedBytes = base64.b64encode(raw_code.encode("utf-8"))
-        encodedStr = str(encodedBytes, "utf-8")
+        encoded_bytes = base64.b64encode(raw_code.encode("utf-8"))
+        encoded_str = str(encoded_bytes, "utf-8")
 
         # create parent directory if not exists
-        self._do_api_call(WORKSPACE_MKDIR_ENDPOINT, {'path': "/Shared/airflow"})
+        path_parts = dbfs_path.split('/')
+        path_parts.pop(0)
+        path_parts = path_parts[:-1]
+
+        path = ''
+        for part in path_parts:
+            path += f'/{part}'
+        self._do_api_call(WORKSPACE_MKDIR_ENDPOINT, {'path': path})
```

Comment:
Would be worth testing how this section behaves on different machines.  I know the os.path lib does a lot of work standardizing this and I've had problem in the past with different file systems misbehaving when manipulating paths without os.path tools.

Reply:
I think we might have a disconnect here. This isn't using the os.path lib, it's just a string. That being said, would it still need to be be tested on different machines?

Reply:
Yeah. I think dbfs path on databricks FS is "standardized" with '/` so there is no need to use pathlib for those.